streaming/java/streaming-runtime/src/main/java/io/ray/streaming/runtime/transfer/DataReader.java

```diff
@@ -246,8 +246,8 @@ private native void getBundleNative(
 
     // kMessageBundleHeaderSize + kUniqueIDSize:
     // magicNum(4b) + bundleTs(8b) + lastMessageId(8b) + messageListSize(4b)
-    // + bundleType(4b) + rawBundleSize(4b) + channelID(20b)
-    static final int LENGTH = 4 + 8 + 8 + 4 + 4 + 4 + 20;
+    // + bundleType(4b) + rawBundleSize(4b) + channelID
+    static final int LENGTH = 4 + 8 + 8 + 4 + 4 + 4 + ChannelId.ID_LENGTH;
```

Comment:
The PR seems to contain some weird thing?

Reply:
This is what's shown in [master](https://github.com/ray-project/ray/blob/4e569ee20bb850315dd836381a790a3e3701437e/streaming/java/streaming-runtime/src/main/java/io/ray/streaming/runtime/transfer/DataReader.java#L250). Should I instead revert back to `static final int LENGTH = 4 + 8 + 8 + 4 + 4 + 4 + 20;`?